client/payment-details/dispute-details/dispute-notice.tsx

```diff
@@ -18,48 +18,52 @@ import { isInquiry } from 'wcpay/disputes/utils';
 
 interface DisputeNoticeProps {
 	dispute: Dispute;
-	urgent: boolean;
+	isUrgent: boolean;
 }
 
 const DisputeNotice: React.FC< DisputeNoticeProps > = ( {
 	dispute,
-	urgent,
+	isUrgent,
 } ) => {
-	const clientClaim =
+	const shopperDisputeReason =
 		reasons[ dispute.reason ]?.claim ??
 		__(
 			'The cardholder claims this is an unrecognized charge.',
 			'woocommerce-payments'
 		);
 
-	const noticeText = isInquiry( dispute )
-		? /* translators: <a> link to dispute documentation. %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
-		  __(
-				// eslint-disable-next-line max-len
-				'<strong>%s</strong> You can challenge their claim if you believe it’s invalid. Not responding will result in an automatic loss. <a>Learn more</a>',
-				'woocommerce-payments'
-		  )
-		: /* translators: <a> link to dispute documentation. %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
-		  __(
-				// eslint-disable-next-line max-len
-				'<strong>%s</strong> Challenge the dispute if you believe the claim is invalid, or accept to forfeit the funds and pay the dispute fee. Non-response will result in an automatic loss. <a>Learn more about responding to disputes</a>',
-				'woocommerce-payments'
-		  );
+	/* translators: <a> link to dispute documentation. %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
+	let noticeText = __(
+		'<strong>%s</strong> Challenge the dispute if you believe the claim is invalid, ' +
+		'or accept to forfeit the funds and pay the dispute fee. ' +
+		'Non-response will result in an automatic loss. <a>Learn more about responding to disputes</a>',
```

Comment:
Split these over multiple lines so we don't need to `eslint-disable-next-line-max-len`. Also for readability.